x-pack/plugins/ml/public/application/components/multi_select_picker/multi_select_picker.tsx

```diff
@@ -26,18 +27,16 @@ export interface Option {
 
 const NoFilterItems = () => {
   return (
-    <div className="euiFilterSelect__note">
-      <div className="euiFilterSelect__noteContent">
-        <EuiIcon type="minusInCircle" />
-        <EuiSpacer size="xs" />
-        <p>
-          <FormattedMessage
-            id="xpack.ml.multiSelectPicker.NoFiltersFoundMessage"
-            defaultMessage="No filters found"
-          />
-        </p>
-      </div>
-    </div>
+    <EuiSelectableMessage>
```

Comment:
@breehall what are the plans for switching out the deprecated `EuiFilterSelectItem` for `EuiSelectable`, as is still being used in our file here?